configurecompiler.cmake

```diff
@@ -509,6 +509,9 @@ if (CLR_CMAKE_PLATFORM_UNIX)
     add_compile_options(-Wno-unused-but-set-variable)
     add_compile_options(-fms-extensions)
     add_compile_options(-Wno-unknown-pragmas)
+    if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7.0)
```

Comment:
This is a short-term solution to be able to compile. We can remove this when @am11 change goes in.

Reply:
This can be kept in a branch without checking it into the master.

Reply:
I am doing that already. I want master to have as little issues as possible.

Since your change is gated with NO_MERGE due to number of changes involved, I want to make forward progress at this moment .

You could remove these lines as part of your change when code becomes compliant.